syntax = "proto3";

service FaceService {
  rpc DetectFacePose (DetectFaceRequest) returns (DetectFaceResponse);
  rpc GenerateModel (GenerateModelRequest) returns (GenerateModelResponse);
}

// SERVICE DETECT FACE

message DetectFaceRequest {
  string image_path = 1;
}

message DetectFaceResponse {
  string result = 1;
}

// SERVICE GENERATE MODEL

message GenerateModelRequest {
  string name_model = 1;
  string type_save = 2;
  string folder_path = 3;
}

message GenerateModelResponse {
  bool success = 1;
}

// SERVICE SCANNING IMAGE IN REAL-TIME
